gcp_variant_transforms/beam_io/vcf_estimate_io.py

```diff
@@ -86,53 +77,69 @@ def __init__(self,
                                             splittable=False)
     self._compression_type = compression_type
 
+  def _get_header_size(self, file_to_read, file_name):
```

Comment:
I am sorry, it is not quite a good idea to have these functions since each of them is kind of still relevant to the others and doing more than one thing. How about combine this and the extract_sample together, and rename it as something like _get_header_size_and_samples or just _get_header_info?